cmake/FindLLVM-Wrapper.cmake

```diff
@@ -23,10 +23,21 @@ if(LLVM-Wrapper_FIND_REQUIRED)
     list(APPEND FIND_ARGS "REQUIRED")
 endif()
 
+set(ALLOWED_LLVM_VERSIONS 15 16 17 18.0 18.1)
```

Comment:
Do we need to specify minor versions of LLVM 18?

Reply:
Yes, LLVM 18.0 and 18.1 are not compatible, searching for 18 will only match on version 18.0 .